### ISSUE TITLE
Is it OK that the training loss kept around 5.8 when steps goes up to 250K？

### ISSUE THREAD

**Issue by butterl** [r9y9/wavenet_vocoder#79]
Hi , I used thchs30 dataset to train wavenet and the loss kept around 5.8 when steps goes up to 200K, I'm not sure if this is normal and if not， any suggestion to make the loss down？

The evaluation wav got may noise in it and if this wave a good one ? ( I think the wav different need to be less， but the below seem not fine)

![step000250000_waveplots](https://user-images.githubusercontent.com/3068190/40766941-a8ca38f4-64e3-11e8-8b1f-31b8d1f5bb0f.png)

**Comment by begeekmyfriend** [comment]
I am afraid you need to wait until 800K steps https://github.com/r9y9/wavenet_vocoder/issues/41

**Comment by butterl** [comment]
Any hparam suggestion？ I only changed the worker num and batch size to fit memory

**Comment by UESTCgan** [comment]
I train it on the data of Ljspeech. After trained over 600k, the loss decrease slowly. I want to know how low can the loss be.
![image](https://user-images.githubusercontent.com/40649244/45744011-ff72d280-bc2f-11e8-873f-1e688ecd25c9.png)
